skins/cat17/address-form/src/types.ts

```diff
@@ -41,4 +42,7 @@ export interface PostData {
     [key: string]: string
 }
 
-
+export interface JQueryTransport {
```

Comment:
Let's lose the `JQuery` here. See the original code where there was a `Transport` interface (although only defined through JSDoc).